src/enzo/Make.mach.sdsc-comet

```diff
@@ -92,8 +96,8 @@ MACH_OPT_AGGRESSIVE  = -O3 -march=native # Flags for aggressive optimization
 # Includes
 #-----------------------------------------------------------------------
 
-LOCAL_INCLUDES_MPI    = 
-LOCAL_INCLUDES_HDF5   = -I$(LOCAL_HDF5_INSTALL)/include # HDF5 includes
+LOCAL_INCLUDES_MPI    =  
+LOCAL_INCLUDES_HDF5   = -I$(LOCAL_MPI_INSTALL)/include -I$(LOCAL_HDF5_INSTALL)/include # HDF5 includes
```

Comment:
Should the MPI includes be moved to the line above?

Reply:
LOCAL_INCLUDES_HDF5 should have -I$(LOCAL_MPI_INSTALL)/include because parallel hdf5 library on comet and this way tools like inits would not compile. So the question is should it be duplicated in LOCAL_INCLUDES_HDF5

Reply:
I wasn't aware that Enzo worked with parallel-enabled hdf5. If that's so, then I think this is just fine. Is there a serial version of hdf5 on Comet?

Reply:
Yes, they do. I tried hdf5-serial/1.10.3 and it works fine. Update Make.mach.sdsc-comet  on that.